jsonexample/test/widget_tests_test.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
```

Comment:
I love that you included tests.

Reply:
I'll be adding some for Shrine, hopefully, in the near future.

Reply:
I'll be writing a few for Shrine in the near future, hopefully.

Reply:
We'll use them too.